coreTable/CoreTable.ts

```diff
@@ -931,7 +931,11 @@ export class CoreTable<
 
     appendRows(rows: ROW_TYPE[], opDescription: string) {
         return this.concat(
-            [new (this.constructor as any)(rows, this.defs) as CoreTable],
+            [
+                new (this.constructor as any)(rows, this.defs, {
+                    parent: this.parent,
```

Comment:
@daniel, this was the reason the derived values were re-calculated.
Without a parent set, `isRoot = true` for the table and it re-applies the transforms. This was happening in the `complete` call of the tolerance implementation.

I don't know if this is the right fix, but it ✨works✨

Reply:
Looks right! Seems like a property that could easily be forgotten.